phpstan.neon.dist

```diff
@@ -7,3 +7,7 @@ parameters:
     paths:
       - lib
       - tests
+    ignoreErrors:
+      # TODO Use newer Doctrine packages which are using T and TKey
```

Comment:
Currently doctrine-website is using outdated collection dependencies where `T` and `TKey` aren't available yet. The outdated version is because doctrine/skeleton-mapper is also using outdated dependencies.